g2p/mappings/langs/font-encodings/fn_unicode.csv

```diff
@@ -1,21 +1,40 @@
+\uF000,x\u030c
 \uF101,c\u0313
-\uF102,č\u0313
+\uF102,c\u030c\u0313
 \uF103,g\u0313
+\uF104,j\u030c\u0313
 \uF105,k\u0313
 \uF106,l\u0315
 \uF107,m\u0313
 \uF108,n\u0313
 \uF109,ŋ\u0313
-\uF110,z\u0313
-\uF111,\u019b\u0313
```

Comment:
These guys were in ASCIIbetical order, whereas the table I've contributed to is ordered by code point.

Reply:
Makes it easier to read, good idea.

Reply:
yep good idea